workbench-cypress/cypress/e2e/pulls/openPulls.spec.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  haveLengthAtLeast,
+  shouldBeVisible,
+  shouldNotExist,
+} from "@sharedTests/sharedFunctionsAndVariables";
+import { newExpectation } from "@utils/helpers";
+import { runTests } from "@utils/index";
+import { selectBranch } from "@utils/sharedTests/changeBranch";
+import { testDBHeader } from "@utils/sharedTests/dbHeaders";
+
+const pageName = "Pull requests page with open pulls";
```

Comment:
We don't have a pull requests list page for the workbench. Maybe:
```suggestion
const pageName = "Open pull page";
```